tests/conftest.py

```diff
@@ -133,3 +136,24 @@ def clean_and_restore_feedbacks(motion_controller):
     mc.configuration.set_velocity_feedback(vel, servo=alias)
     mc.configuration.set_position_feedback(pos, servo=alias)
     mc.configuration.set_auxiliar_feedback(aux, servo=alias)
+
+
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call):
+    # execute all other hooks to obtain the report object
+    outcome = yield
+    rep = outcome.get_result()
+
+    # store test results for each phase of a call, which can be "setup", "call", "teardown"
+    item.stash.setdefault(test_report_key, {})[rep.when] = rep
```

Comment:
Why you use `item.stash.setdefault(test_report_key, {})[rep.when] = rep` instead `item.stash[test_report_key] = {rep.when: rep}`?

Reply:
I just followed this code: https://docs.pytest.org/en/latest/example/simple.html#making-test-result-information-available-in-fixtures